examples/fluids/src/grid_anisotropy_tensor.c

```diff
@@ -124,6 +121,7 @@ PetscErrorCode GridAnisotropyTensorProjectionSetupApply(Ceed ceed, User user, Ce
   // -- Cleanup
   PetscCall(NodalProjectionDataDestroy(grid_aniso_proj));
   PetscCall(OperatorApplyContextDestroy(l2_rhs_ctx));
+  PetscCall(OperatorApplyContextDestroy(mass_matop_ctx));
```

Comment:
I'm scanning for differences - this line was added but doesn't seem to be part of the main thrust of this PR?

Reply:
Correct. It came about because I found several leaks/unused variables in the other function in the file. I've tried with and without the changes, but the test results are still the same.

Reply:
Sounds good to me. #1266